shepherd/Goal.py

```diff
@@ -47,13 +48,23 @@ def score(self, alliance):
             scorring Alliance is the owning alliance.
                 alliance - Alliance object representing the scoring team
         """
-        pass
+        if self.owner is alliance:
+            alliance.change_score(self.value * self.calc_multiplier() *\
+                alliance.alliance_multiplier)
 
     def calc_multiplier(self):
         """ returns an integer that represents the overall multiplier for this
             goal, by checking each timer in the goal
         """
-        pass
+        if (self is None or self.gold_two_x_timer is None or
```

Comment:
How could self be None. Also, is this check even necessary? I think the second case does/should just handle no active timers.